types/webpack/index.d.ts

```diff
@@ -349,14 +350,16 @@ declare namespace webpack {
     type ExternalsFunctionElement = (context: any, request: any, callback: (error: any, result: any) => void) => any;
 
     interface Node {
-        console?: boolean;
+        console?: boolean | 'mock';
```

Comment:
why this and `process` can't be `"empty"`?

Reply:
https://webpack.js.org/configuration/node/#node-console
https://webpack.js.org/configuration/node/#node-process
🤔

Reply:
see https://webpack.js.org/configuration/node/#node-process

try it
```
node: {
        process: 'empty'
}

npm run build
```

Output result
```
> webpack --config webpack.development.js

Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.node should be one of these:
   false | object { Buffer?, __dirname?, __filename?, console?, global?, process?, … }
   -> Include polyfills or mocks for various node stuff.
   Details:
    * configuration.node should be false
    * configuration.node.process should be one of these:
      false | true | "mock"
      -> Include a polyfill for the 'process' variable
```